### PR TITLE
Add Pod Dashboard

### PR DESCRIPTION
**Pull request by mareklibra**
So far with just a single `Details` card.
Additional cards will be added in follow-ups.

Depends on: 
- [x] #2600
- [x] #2809 

---
podDashboard

### CODE REVIEW

**Comment by mareklibra** [comment]
Rebased as dependencies are merged now.
@vojtechszocs , can you please have a look?

File: frontend/public/components/pod-dashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { PodKind } from '../module/k8s';
+import { Dashboard, DashboardGrid } from './dashboard';
+import { PodDashboardDetailsCard } from './pod-dashboard-details';
+
+export const PodDashboardContext = React.createContext<PodDashboardContext>({});
```
Comment (mareklibra) [inline frontend/public/components/pod-dashboard.tsx]: Will be extracted to separate file, so will be similar to vm-dashboard-context

Comment (mareklibra) [reply frontend/public/components/pod-dashboard.tsx]: done

**Comment by spadgett** [comment]
As a rule, we try to keep master as close to production as possible and only enable features in the UI when they're ready (i.e., if we had to ship tomorrow, does it make sense to ship this as-is?). In this case, I would add the code, but hide the tab until there is more content.

**Pull request merged**